benchmarks/vacancy_migration/run.py

```
@@ -36,16 +34,11 @@ def save_to_pickle(
     #     json.dump(result, f)
 
+
 @task
 def calculate_vacancy_migration(
-    pristine: Atoms,
-    istart: int,
-    iend: int,
-    calculator_name: MLIPEnum | str,
-    optimizer: str,
-    criterion: dict = {}
+    pristine: Atoms, istart: int, iend: int, calculator_name: MLIPEnum | str, optimizer: str, criterion: dict = {}
```

Ruff (0.15.9)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)